src/components/badge/badge.scss

```diff
@@ -5,21 +5,28 @@
  * @prop --badge-text-color: badge text color
  */
 
-
 :host {
-  display: block;
+    display: block;
+}
+
+:host([hidden]) {
+    display: none;
+}
+
+* {
```

Comment:
If I may ask, why didn't you add the setting `box-sizing` in `.badge-container` class? I just wondered

Reply:
Two reasons:

- Mainly to make it easy to copy and paste to other files. (I'd like to have this in every component, but I'm not sure if doing that might screw up the stylings we get from MDC.)
- Secondarily, just in case we add more elements some time in the future, not applying the same box-sizing to all elements in the component could be extremely confusing.

Reply:
You may always ask! 😉
Asking is always good 👍